transforms/.make.cicd.targets

```diff
@@ -7,10 +7,15 @@
 include $(REPOROOT)/transforms/.make.transforms
 
 ######################################################################
-## Default setting for TRANSFORM_RUNTIME uses folder name-- Old layout
-TRANSFORM_PYTHON_RUNTIME_SRC_FILE=-m dpk_$(TRANSFORM_NAME).transform
-TRANSFORM_RAY_RUNTIME_SRC_FILE=-m dpk_$(TRANSFORM_NAME).ray.transform
-TRANSFORM_PYTHON_RUNTIME_SRC_FILE=-m dpk_$(TRANSFORM_NAME).spark.transform
+## Default setting for TRANSFORM_RUNTIME entry point: 
+# python -m dpk_html2parquet.ray.transform --help
```

Comment:
do we really need the `dpk` prefix for each transformer? If you want to prevent possible name conflicts, what about ` dpk` namespace? e.g. `dpk.html2parquet.ray.transform`

Reply:
@roytman : @dolfim-ibm made a passionate argument for this. I am good either way. @dolfim-ibm @daw3rd Can yo guys weigh in ? thanks

Reply:
my argument was about having something, i.e. either prefix or namespace. I think it is very important to have a library which doesn't provide lots of global names which might overlap with others.
it also brings a bit of "branding" and makes it clearer to users that all these transforms come from the same place, i.e. data-prep-kit.

Reply:
It's just a suggestion; I'm OK with any decision.